app/full_text_search.py

```diff
@@ -113,52 +116,93 @@ def search(repo, query_txt, max_results):
     return results
 
 
-def create_jp_name_fields(**kwargs):
+def create_full_name_without_space(name_component1, name_component2):
     """
-    Creates fields(romanized_jp_names) for full text search.
+    Creates full name without white space.
+    Returns:
+        if name_component1 and name_component2: 'name_component1 + name_component2'
+        else: None
+    """
+    full_name_without_space = ''
+    if name_component1 and name_component2:
+        full_name_without_space = (name_component1 + name_component2,
+                                   name_component2 + name_component1)
+    return full_name_without_space
+
+
+def create_full_name_without_space_fields(romanize_method, given_name, family_name):
+    """
+    Creates fields with the full name without white spaces.
+    Returns:
+        fullname fields, romanized_name_list: (for check)
+    """
+    fields = []
+    romanized_name_list = []
+    romanized_given_name = romanize_method(given_name)
+    romanized_family_name = romanize_method(family_name)
+    full_names = create_full_name_without_space(
+        romanized_given_name, romanized_family_name)
+    if full_names:
+        full_name_given_family = full_names[0]
```

Comment:
Optional but this may be simpler:

    full_name_given_family, full_name_family_given = full_names